### PR TITLE
Update advance section

### PR DESCRIPTION
**Pull request by priyanshupardhi**
Added localstorageVSsessionstorage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ https://annu12340.github.io/Javascript-Notes/
 
 <br/>
 
+- Local storage vs Session storage
```
Comment (annu12340) [inline README.md:35]: Could add this heading on line 29 (because after that <br/> on line 30, includes more advanced topics)

File: docs/README.md [hunk 2]
```diff
@@ -43,13 +43,10 @@
 - throttle vs debounce
 - object creation patterns tutorial - factory , constructor pattern, prototype pattern
 - JavaScript ES2020
+- Local storage vs Session storage
 - Tricky JavaScript interview questions and answers
 
 
-### Good references
-
-1. https://medium.com/javascript-scene/10-interview-questions-every-javascript-developer-should-know-6fa6bdf5ad95
```
Comment (annu12340) [inline docs/README.md:59]: I think you accidentally deleted this line (pls include it)

File: docs/README.md [hunk 3]
```diff
@@ -807,7 +804,25 @@ The new JavaScript features in ES2020 are:
...
+localStorage and sessionStorage accomplish the exact same thing and have the same API, but with sessionStorage the data is persisted only until the window or tab is closed, while with localStorage the data is persisted until the user manually clears the browser cache or until your web app clears the data. The examples in this post are for localStorage, but the same syntax works for sessionStorage.
+
+```
+Create key/value pair entries with localStorage.setItem, providing a key and a value:
+
+let key = 'Item 1';
+localStorage.setItem(key, 'Value');
+
+``` 
+
```
Comment (annu12340) [inline docs/README.md:1149]: looks good

**Review by annu12340 — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 2 above)
Comment (priyanshupardhi) [reply docs/README.md:59]: okay will fix it

File: README.md (hunk 1 above)
Comment (priyanshupardhi) [reply README.md:32]: okay, just heading or the whole definition part

Comment (priyanshupardhi) [reply README.md:35]: Okay

**Comment by annu12340** [comment]
Pls resolve the conflicts

**Comment by priyanshupardhi** [comment]
@annu12340 have a look

File: docs/README.md [hunk 4]
```diff
@@ -1136,25 +1135,6 @@ _Reference_
 - https://radiant-brushlands-42789.herokuapp.com/medium.com/better-programming/javascript-es2020-features-with-simple-examples-d301dbef2c37
 - https://www.freecodecamp.org/news/javascript-new-features-es2020/
 
-<br/> 
-
-## 17. Local storage vs Session storage
-
```
Comment (annu12340) [inline docs/README.md:1144]: Hey why did you remove this content?
Pls add it back and then I could merge the pr

**Review by annu12340 — CHANGES_REQUESTED** [review]

**Comment by annu12340** [comment]
@priyanshupardhi are you facing any difficulties with this topic?

**Comment by annu12340** [comment]
👍
